documentation.md

```diff
@@ -733,3 +733,25 @@ boxoffice = imdb.BoxOffice()
 | Methods         | Details                                                                       |
 | --------------- | ----------------------------------------------------------------------------- |
 | `.top_movies()` | Returns the top box office movies, weekend and total gross and weeks released |
+
```

Comment:
Add the documentation part in `dev-documentation.md` not in the `documentation.md` file.